src/sage/rings/qqbar.py

```diff
@@ -2508,6 +2509,8 @@ def number_field_elements_from_algebraics(numbers, minimal=False, same_field=Fal
     - ``embedded`` -- Boolean (default: ``False``). Whether to make the
       NumberField embedded.
 
+    - ``name`` -- string (default: ``a``). Name of the primitive element.
+
```

Comment:
```suggestion
    - ``name`` -- string (default: ``'a'``); name of the primitive element
```
Let's not make things worse; instead, let's bring everything into the correct format.